DataFormats/TrackerRecHit2D/BuildFile.xml

```diff
@@ -7,6 +7,7 @@
 <use name="FWCore/MessageLogger"/>
 <use name="FWCore/Framework"/>
 <use name="Geometry/CommonDetUnit"/>
+<use name="Geometry/TrackerGeometryBuilder"/>
```

Comment:
this looks like a quite undesired new dependency (it adds 3-4 new packages outside of DataFormats )

Reply:
Turns out this is not needed, I'll remove it.